src/main/java/org/jahia/modules/sitemap/services/SitemapServiceImpl.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * ==========================================================================================
+ * =                            JAHIA'S ENTERPRISE DISTRIBUTION                             =
+ * ==========================================================================================
+ *
+ *                                  http://www.jahia.com
+ *
+ * JAHIA'S ENTERPRISE DISTRIBUTIONS LICENSING - IMPORTANT INFORMATION
+ * ==========================================================================================
+ *
+ *     Copyright (C) 2002-2021 Jahia Solutions Group. All rights reserved.
+ *
+ *     This file is part of a Jahia's Enterprise Distribution.
+ *
+ *     Jahia's Enterprise Distributions must be used in accordance with the terms
+ *     contained in the Jahia Solutions Group Terms &amp; Conditions as well as
+ *     the Jahia Sustainable Enterprise License (JSEL).
+ *
+ *     For questions regarding licensing, support, production usage...
+ *     please contact our team at sales@jahia.com or go to http://www.jahia.com/license.
+ *
+ * ==========================================================================================
+ */
+package org.jahia.modules.sitemap.services;
+
+import net.htmlparser.jericho.Source;
+import org.jahia.modules.sitemap.exceptions.SitemapException;
+import org.osgi.service.component.annotations.Component;
+
+import org.jahia.modules.sitemap.config.ConfigService;
+import org.osgi.service.component.annotations.Reference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.List;
+
+@Component(immediate = true, service = SitemapService.class)
+public class SitemapServiceImpl implements SitemapService {
+
+    private static Logger logger = LoggerFactory.getLogger(SitemapServiceImpl.class);
+    private static String CHAR_ENCODING="ISO-8859-1";
+
+    private static final String ERROR_IO_EXCEPTION_WHEN_SENDING_URL_PATH = "Error IO exception when sending url path";
+
+    private ConfigService configService;
+
+    @Reference(service = ConfigService.class)
+    public void setConfigService(ConfigService configService) {
+        this.configService = configService;
+    }
+
+    @Override
+    public Boolean sendSitemapXMLUrlPathToSearchEngines(String sitemapUrlPath) throws SitemapException {
```

Comment:
I read somewhere that if you're simply return a primitive type, in this case a boolean, it's better to use that primitive type instead - i.e instead of `public Boolean sendSitemapXMLUrlPathToSearchEngines` use `public boolean sendSitemapXMLUrlPathToSearchEngines`

however, since I don't have much context on this, you can ignore this comment if there's an argument to be made to use the box type instead.

Reply:
👍🏻

Reply:
👍